app/javascript/toggle_action.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener("turbolinks:load", function() {
```

Comment:
sao phải dùng "turbolinks:load" ? mấy file trước không dùng sao file này dùngsao phải dùng "turbolinks:load" ? mấy file trước không dùng sao file này dùng

Reply:
Dạ tại e thấy trong rails có hỗ trợ turbolink trong rails nên e sd thử, với e đọc thì thấy sd nó sẽ nhanh và thuận tiện hơn như là chỉ cần load lại 1 phần trang nếu có sự thay đổi mà không cần load lại toàn trang

Reply:
bỏ turbolink đi nha em, nếu dùng thì dùng hết cho toàn bộ file js còn ko dufg thì thôi, hiện tại với turbolink thì rails đã bị lỗi lưu cache url sau khi submit form fail mà reload lại page thì sẽ chết trang nha

Reply:
chưa fix tuboload nè